Projections/Benchmark/Benchmark.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
+    <Platforms>x64;x86</Platforms>
+    <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
```

Comment:
redundant with Directory.Build.Props, if you want a single place to update all

Reply:
Removed it for all projection csprojs.